doc/api/esm.md

```diff
@@ -94,10 +94,8 @@ until the root of the volume is reached.
 }
 ```
 
-```bash
-# In same folder as above package.json
-node my-app.js # Runs as ES module
-```
+If in the same folder as the above package.json, `node my-app.js` runs
+`my-app.js` as an ES module.
```

Comment:
in this example, the command no longer stands out with this change, and I find it less readable. why don't we want syntax highlighting on command line commands?